### ISSUE TITLE
Incorrect syntax for '= default;' and '= delete;'

### ISSUE THREAD

**Issue by gtianw** [jeff-hykin/better-cpp-syntax#587]
The syntax about `= default;`/`= delete;`, in copy/move assignment operator, are incorrect.

#496 has reported a bug of deleted function.

But it also happen to default function. See microsoft/vscode-cpptools#9281  for details.

**Comment by EloCode** [comment]
Probably the same cause in
```
class Base {
    virtual ~Base() = default;
}

class Derived:Base {
    ~Derived() override = default;
}
```
the override keyword is not highlighted at all. There is no info regarding "semantic tokens". If `= default ` is removed, highlighting works again. 
![tmp2](https://user-images.githubusercontent.com/52861757/185691753-8ad1e30b-5a4c-4f05-8331-2ecc10d71b01.PNG)

**Comment by sean-mcmanus** [comment]
This also repros for "= 0" (abstract virtual method) see https://github.com/microsoft/vscode-cpptools/issues/9957.

**Comment by jeff-hykin** [comment]
Should be fixed now (`v1.15.24` released just now), but let me know if it persists
